contracts/src/xchain/XRegistry.sol

```diff
@@ -25,7 +25,7 @@ contract XRegistry is Ownable, XRegistryBase {
     /**
      * @notice OmniPortal contract.
      */
-    IOmniPortal public omni;
+    OmniPortal public omni;
```

Comment:
why chang from an interface?

Reply:
Need `OmniPortal.initSourceChain`, which I didn't think we needed to add to "public facing" omni portal interface